### PR TITLE
Update integrations

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: docs/documentation.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+:noindex:
+
+Documentation
+=================
+
+Target format and Airmass
+^^^^^^^^^^^^^^^^^^^^^^^^^
+.. automodule:: rise_set.astrometry
```
Comment (eheinrich) [inline docs/documentation.rst:8]: The content for the "Target format and Airmass" and "Visibility API and Example Usage" sections aren't showing up in readthedocs... I think maybe these have to be `.. automodule:: astrometry` and `.. automodule:: visibility`

File: .coveragerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[run]
+source = schedule
```
[the remark this answers is not in the document]
Comment (jnation3406) [reply .coveragerc:2]: haha i copied that and didn't change, oops

File: docs/documentation.rst (hunk 1 above)
Comment (eheinrich) [reply docs/documentation.rst:8]: Actually, I don't think this works

File: rise_set/astrometry.py [hunk 3]
```diff
@@ -9,10 +7,39 @@
 The code uses SLALIB for the heavy lifting. The rise/set/transit algorithms here
 are implementations of Astronomical Algorithms, Ch. 14 (Jean Meeus).
 
-
-Author: Eric Saunders (esaunders@lcogt.net)
-May 2010
-'''
+The input to most astrometry calculation involves a target dictionary containing the
+necessary target data. This package contains functions to help creating a target
```
Comment (eheinrich) [inline rise_set/astrometry.py:11]: "This package contains functions for creating a target dictionary and validating required fields for the supported target types."

File: rise_set/visibility.py [hunk 4]
```diff
@@ -94,11 +119,15 @@ def __init__(self, site, start_date, end_date, horizon=0, twilight='sunrise',
...
-        '''
-
+        """Returns the dark intervals for the site.
+        
+        Returns the night time dark intervals for the given site and date range set
+        in this visibility object.
+
+        Returns:
+            list: A list of tuples of start/end datetime pairs that make up the dark
+                  intervals for this site.
```
Comment (eheinrich) [inline rise_set/visibility.py:129]: This part is showing up with some odd spacing in the generated docs

File: rise_set/visibility.py [hunk 5]
```diff
@@ -111,11 +140,15 @@ def get_dark_intervals(self):
...
-        '''
-
+        """Returns the dark moon intervals for the site.
+        
+        Returns the dark moon intervals (moon is not visible) for the given
+        site and date range set in this visibility object.
+
+        Returns:
+            list: A list of tuples of start/end datetime pairs that make up
+                  the dark moon intervals for this site.
```
Comment (eheinrich) [inline rise_set/visibility.py:150]: Same weird spacing

**Pull request merged**